src/Form/EmbedButtonForm.php

```diff
@@ -73,6 +73,23 @@ public function form(array $form, FormStateInterface $form_state) {
     $upload_directory = \Drupal::config('entity_embed.settings')->get('upload_directory');
     $upload_location = $file_scheme . '://' . $upload_directory . '/';
 
+    $entity_types = $this->entityManager->getEntityTypeLabels(TRUE);
+    $filtered_entity_types = array();
+    // Add all Content entites by default.
+    $filtered_entity_types['Content'] = $entity_types['Content'];
```

Comment:
Those groups are translatable strings, so that won't work. Maybe rely on the currently official but complicated method with isSubclassOf, see https://www.drupal.org/node/2194783?

And it might be easier to just build the list yourself with getDefinitions(), then you can for example also call $entity_type->hasViewBuilder().